sites-enabled/10-www.freecodecamp.org.conf

```diff
@@ -163,10 +163,10 @@ server {
   }
 
   # catch common subpath for langs that we have special configs for
-  location ~ ^/spanish/?(.*)$ {
+  location ~ ^/spanish/(.*)$ {
     return 302 https://www.freecodecamp.org/espanol/$1;
   }
-  location ~ ^/chinese/?(.*)$ {
+  location ~ ^/chinese/(.*)$ {
```

Comment:
Does this not mean that `freecodecamp.org/chinese` will **not** be redirected, when it should?

Reply:
Good question, I'll check.

Reply:
I'm glad I you asked... let me try again.